db/migrate/20160707064120_remove_elections_posts.rb

```diff
@@ -0,0 +1,12 @@
+class RemoveElectionsPosts < ActiveRecord::Migration
+  def up
+    drop_table(:elections_posts)
```

Comment:
This is the old `has_and_belongs_to_many` table, the new one is `election_posts` without plural on elections.